packages/manager/modules/pci/src/projects/project/instances/instances.html

```diff
@@ -129,88 +136,108 @@
+                ></span>
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{:: 'pci_projects_project_instances_rescueEnd_label' | translate }}"
                 data-on-click="$ctrl.endRescueInstance($row)"
                 data-ng-if="$row.status === 'RESCUE'"
             >
+                <span
+                    data-translate="pci_projects_project_instances_rescueEnd_label"
+                ></span>
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{:: 'pci_projects_project_instances_softReboot_label' | translate }}"
                 data-on-click="$ctrl.softRebootInstance($row)"
                 data-ng-if="$row.status === 'ACTIVE'"
             >
+                <span
+                    data-translate="pci_projects_project_instances_softReboot_label"
+                ></span>
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{:: 'pci_projects_project_instances_hardReboot_label' | translate }}"
                 data-on-click="$ctrl.hardRebootInstance($row)"
                 data-ng-if="$row.canBeHardRebooted()"
             >
+                <span
+                    data-translate="pci_projects_project_instances_hardReboot_label"
+                ></span>
             </oui-action-menu-item>
             <oui-action-menu-divider></oui-action-menu-divider>
             <oui-action-menu-item
-                data-text="{{:: 'pci_projects_project_instances_resume_label' | translate }}"
                 data-on-click="$ctrl.resumeInstance($row)"
                 data-ng-if="$row.status === 'SUSPENDED'"
             >
+                <span
+                    data-translate="pci_projects_project_instances_resume_label"
+                ></span>
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{:: 'pci_projects_project_instances_reinstall_label' | translate }}"
                 data-on-click="$ctrl.reinstallInstance($row)"
                 data-ng-if="$row.status === 'ACTIVE'"
             >
+                <span
+                    data-translate="pci_projects_project_instances_reinstall_label"
+                ></span>
             </oui-action-menu-item>
-            <oui-action-menu-item
-                data-text="{{:: 'pci_projects_project_instances_delete_label' | translate }}"
-                data-on-click="$ctrl.deleteInstance($row)"
-            >
+            <oui-action-menu-item data-on-click="$ctrl.deleteInstance($row)">
+                <span
+                    data-translate="pci_projects_project_instances_delete_label"
+                ></span>
             </oui-action-menu-item>
         </oui-action-menu>
 
         <extra-top>
             <a
-                data-ui-sref="pci.projects.project.instances.add"
+                data-ng-click="$ctrl.addInstance()"
```

Comment:
It  would be better if it was a `button` I think

Reply:
This is done. Thanks :)